integration/integration.go

```diff
@@ -248,3 +233,38 @@ func (i *Integration) checkArguments() error {
 
 	return errors.New("arguments must be a pointer to a struct (or nil)")
 }
+
+func (i *Integration) getAnonymousEntity() *Entity {
```

Comment:
instead of this approach, you could add the anonymous entity inside the integration 
type Integration struct {
anonEntity Entity,
....
}
and then you add the metrics, events and inventory to that entity. Before creating the json, you can append that anonymous entity to the entity list (and remove it after the creating the json)

Reply:
good idea. probably safer to do that way